test/issue1504-envvar-in-path.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+
+rm -rf ${CURR_DIR}/issue1504-envvar-in-path/.dub
+rm -rf ${CURR_DIR}/issue1504-envvar-in-path/test
+rm -rf ${CURR_DIR}/output-1504.txt
+
+
+export MY_VARIABLE=teststrings 
+# pragma(msg) outputs to stderr
+${DUB} build --force --root ${CURR_DIR}/issue1504-envvar-in-path 2> ${CURR_DIR}/output-1504.txt
+
+grep "env_variables_work" < ${CURR_DIR}/output-1504.txt
+
```

Comment:
The temporary files should have been removed here :/

Reply:
They are removed before the test, like in the other test about variables.